src/OmniSharp.DotNetTest/TestManager.cs

```diff
@@ -76,6 +76,11 @@ public static TestManager Create(Project project, DotNetCliService dotNetCli, IE
         public abstract GetTestStartInfoResponse GetTestStartInfo(string methodName, string testFrameworkName, string targetFrameworkVersion);
 
         public abstract Task<DebugTestGetStartInfoResponse> DebugGetStartInfoAsync(string methodName, string testFrameworkName, string targetFrameworkVersion, CancellationToken cancellationToken);
+
+        public virtual Task<DebugTestGetStartInfoResponse> DebugGetStartInfoAsync(string[] methodNames, string testFrameworkName, string targetFrameworkVersion, CancellationToken cancellationToken)
```

Comment:
What other types derive from TestManager? Should they implement this?

Reply:
There's just two -- a legacy one for the old 'dotnet test' on project.json projects and the current 'dotnet vstest' version. I would only expect us to make this work for the newer stuff.

Reply:
I have implemented this function only for the VSTestManager and not LegacyTestManager. Is that required ?

Reply:
What is the behavior if you open a project with legacy tests? Does the `run tests` button show up? If it does, and you click it, what happens?

Reply:
A project with legacy tests == a project.json project. I don't think we should invest effort in that. So long as we don't break the ability to run/debug a single test in a project.json project, I don't think we need to make any changes to `LegacyTestManager`.

Reply:
Sounds good. I'm happy with our solution of not showing the indicator for legacy projects.